### ISSUE TITLE
"New Terminal" shortcut key should be replaced

### ISSUE THREAD

**Issue by jooyoungseo** [rstudio/rstudio#6931]
### System details

    RStudio Edition : Server
    RStudio Version : 1.3.959
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem

1. Go to "Tools" menu in menubar by pressing `"alt+Shift+S"` on Windos or Linux; `"Ctrl+Opt+L"` on Mac.
2. Find and expand "Terminal" menu.
3. Check "New Terminal" item with its assigned shortcut key.

### Describe the problem in detail

On Windows and Linux, `"Alt+Shift+"` key combination have been assigned towards menubar access (e.g., `"Alt+Shift+F"` for File; `"Alt+Shift+H"` for Help, etc.).

It would be safer to not preserve "Alt+Shift" key combination for "New Terminal" for the sake of logical consistency.

### Describe the behavior you expected

Please consider replacing `"Alt+Shift+R"` for "New Terminal" not  using "Alt+Shift" key convention.

- [x] I have read the guide to submitting good bug reports at https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports . 
- [x] I have installed the latest version of RStudio and confirmed that the issue still persists.
- [x] If I am reporting a RStudio crash, I have included a diagnostics report. https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by gtritchie** [comment]
Quite a few shortcuts using the `Alt+Shift` prefix existed before I started using it for the main menu. An example is `Alt+Shift+G` for "Goto Line".

So for now I'm going to keep these the same, I have made adjustments so the main menu shortcuts are now more logical in #7061.

Closing as "won't fix".
